server/views/pages/bookedtoday/choosePrisoner.njk

```diff
@@ -1,10 +1,11 @@
 {% extends "../../partials/layout.njk" %}
 {% from  "../../partials/expectedArrivalCardMacro.njk" import expectedArrivalCard, transferCard  %}
-{% set pageTitle = applicationName + " - Home" %}
+{% set pageTitle = "Prisoners booked to arrive today" if supportingMultitransactionsEnabled == true else "Select prisoner to add to the establishment roll" %}
```

Comment:
I think the previous content of `application + " - Home"` was a hangover from when this page was effectively the app home page so updated `pageTitle` and used this as the title variable as done on the temporary absences page. Flagging in case the browser tab text should be reverted back to `application + " - Home"`.